INSTALL.rst

```diff
@@ -99,6 +131,7 @@ setuptools to latest too:
     (inspirehep)$ pip install --upgrade pip setuptools
     (inspirehep)$ cdvirtualenv src/inspirehep
     (inspirehep)$ pip install -r requirements.txt --pre --exists-action i
+    (inspirehep)$ pip install honcho
```

Comment:
Shall we add ``honcho`` as a dependency in the ``webserver``? https://github.com/inspirehep/inspire-next/blob/master/setup.py#L133 :)